Runtime/Data/CoreSettings.cs

```diff
@@ -4,11 +4,17 @@ namespace ReadyPlayerMe.Core.Data
 {
     public class CoreSettings : ScriptableObject
     {
+        private const string RESOURCE_PATH = "Settings/CoreSettings";
         public const string DEFAULT_SUBDOMAIN = "demo";
 
         public string Subdomain = DEFAULT_SUBDOMAIN;
         public string AppId = string.Empty;
         public bool EnableLogging;
         public bool EnableAnalytics;
+
+        public static CoreSettings Load()
+        {
+            return Resources.Load<CoreSettings>(RESOURCE_PATH);
+        }
```

Comment:
What do you think of moving this to handler, so that this script is just the data?

Reply:
sure can do that. 
to me it's a little more convenient to be able to go CoreSettings.Load() as a first-time developer wanting to know how I can get the core settings. Instead of having to know there is a separate CoreSettingsHandler class for loading. 
I understand though that CoreSettings should be just a data object. 
The other thing is that in this sense the handler should be renamed something like CoreSettingsRuntimeLoader or something and the name change will cause a breaking change. 

in any case I will move it back to handler :D

Reply:
updated. will merge